backend/src/email/services/email-transactional.service.ts

```diff
@@ -262,6 +263,11 @@ async function handleStatusCallbacks(
   }
 }
 
+export type EmailMessageTransactionalWithRelations =
```

Comment:
Hey u dont need a new type, just add this field to the model and annotate it as a relational column